stepprogress/src/main/java/com/funrisestudio/stepprogress/StepProgressView.kt

```diff
@@ -505,6 +505,63 @@ class StepProgressView : ViewGroup, View.OnClickListener {
         return stepProgress.isAllDone()
     }
 
+    /**
+     * Change title text padding
+     */
+    fun setTextTitlePadding(size: Float) {
+        textTitlePadding = size
+        requestLayout()
+    }
+
+    /**
+     * Change node title text size
+     */
+    fun setNodeTitleSize(size: Int) {
+        textNodeTitleSize = size
+        resetView()
+        invalidate()
+    }
+
+    /**
+     * Change text node  size
+     */
+    fun setNodeTextSize(size: Int) {
+        textNodeSize = size
+        requestLayout()
```

Comment:
It's better to call resetView() instead of requestLayout() so that you'll be able to call this method in runtime to change textSize

Reply:
fixed